frontend/encore/dev-server.rst

```diff
@@ -69,50 +103,17 @@ method in your ``webpack.config.js`` file:
 Hot Module Replacement HMR
 --------------------------
 
-Encore *does* support `HMR`_ for :doc:`Vue.js </frontend/encore/vuejs>`, but
-does *not* work for styles anywhere at this time. To activate it, pass the ``--hot``
-option:
-
-.. code-block:: terminal
-
-    $ ./node_modules/.bin/encore dev-server --hot
-
-If you want to use SSL with self-signed certificates, add the ``--https``,
-``--pfx=``, and  ``--allowed-hosts`` options to the ``dev-server`` command in
-the ``package.json`` file:
-
-.. code-block:: diff
+Hot module replacement is a superpower of the ``dev-server`` where styles and
+(in some cases) JavaScript can automatically update without needing to reload
+your page. HMR works automatically with CSS (as long as you're using the
```

Comment:
Oh! Does it means we don't have to do `Encore.disableCssExtraction(Encore.isDevServer())`? Nice! :)

Reply:
I haven't read about this specifically, but when I tested it on my project earlier (which DOES always use CSS extraction), it totally worked!

Reply:
I confirm I didn't had to use `Encore.disableCssExtraction(Encore.isDevServer())` on my project too!